app/src/main/res/values/strings.xml

```diff
@@ -633,6 +633,10 @@
   <string name="wallet_promotion_wallet_notinstalled_message">Install this app and get %s APPC Credits. You need the AppCoins Wallet to receive them.</string>
   <string name="wallet_promotion_install_button">Install Wallet</string>
   <string name="wallet_promotion_no_button">No, thanks</string>
+  <string name="wallet_promotion_button_install_disabled">Install to get %s APPC Credits</string>
```

Comment:
Shouldn't these strings be added by Carlos? Did these strings went through the correct process?

Reply:
He knows